src/main/java/TulipTask.java

```diff
@@ -0,0 +1,131 @@
+import java.util.HashMap;
+import java.util.Scanner;
+
+public class TulipTask {
+    public static void main(String[] args) throws TulipTaskException.InvalidTaskDescriptionException, TulipTaskException.InvalidEndDateException, TulipTaskException.InvalidStartDateException, TulipTaskException.InvalidDeadlineException, TulipTaskException.InvalidTaskIndexException {
```

Comment:
The main method seems quite long, maybe you could refactor it to several smaller methods!